index.d.ts

```diff
@@ -683,56 +718,78 @@ declare module 'plaid' {
 
     getItem: AccessTokenFn<ItemResponse>;
 
-    updateItemWebhook(accessToken: string,
-                      webhook: string,
+    importItem(
+      products: Array<string>,
+      userAuth: Map<string,string>,
+      options?: WebhookOptions,
+    ): void;
+    importItem(
+      products: Array<string>,
+      userAuth: Map<string,string>,
+      cb: Callback<TokenResponse>,
```

Comment:
`s/Callback<TokenResponse>/Callback<ImportItemResponse>` unless you meant to re-use `TokenResponse` here